.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+<!--
+Before making a PR, please read our contributing guidelines https://github.com/operator-framework/operator-sdk/blob/master/CONTRIBUTING.MD
```

Comment:
Should we ask submitters to make sure their branches are rebased to the latest revision of the destination branch before submitting?

Reply:
I would say yes. Especially given that we vendor the contributor's branch as the source in our e2e test.

If their branch is significantly behind the master there can be conflicts.

Reply:
Yeah, that would be useful considering that the tests tend to fail a lot simply due to the way we have to handle some of our tests due to dep.